detect_variants/vep.cwl

```diff
@@ -3,22 +3,20 @@
 cwlVersion: v1.0
 class: CommandLineTool
 label: "Ensembl Variant Effect Predictor"
-baseCommand: ["/usr/bin/perl", "/usr/bin/variant_effect_predictor.pl"]
-
+baseCommand: ["/home/vep/src/ensembl-vep/vep", "-I", "/home/vep/Plugins"]
 requirements:
+    - class: ShellCommandRequirement
```

Comment:
(I see it's in the `toil_compatibility` version, too, but) what's using the `ShellCommandRequirement`?

Reply:
I'm always confused when to use which requirements. I think I might've added it for the `cache_dir` and `hgvs` `valueFrom` inputs. But I guess it's not needed for that?

Reply:
Yeah, the `valueFrom` expressions are the `InlineJavascriptRequirement`.  The most common reason we need `ShellCommandRequirement` is for pipes.

Reply:
Ok, I removed it.